lib/expeditor/command.rb

```diff
@@ -37,18 +45,18 @@ def start_with_retry(retryable_options = {})
     end
 
     def started?
-      @normal_future.executed?
+      !!@normal_future && @normal_future.executed?
```

Comment:
>  should be hidden from library users.

I agree it, but `@normal_future && @normal_future.executed?` doesn't returns `@normal_future`, does it?
Although it might returns `nil`, `nil` does not show any information about `@normal_future`.